Makefile

```diff
@@ -307,6 +307,17 @@ remove_lib_wrapped:
 remove_locales_wrapped:
 	for f in $(LANGUAGES); do $(RM) $(DESTDIR)$(LOCALEDIR)/$$f/LC_MESSAGES/$(PACKAGE).mo; done
 
+test:
+	@echo "Running tests..."
+	# Set LD_LIBRARY_PATH to include the current directory
+	@export LDFLAGS="-Wl,-rpath,$(CURDIR)"; \
+	export LD_LIBRARY_PATH=$(CURDIR):$$LD_LIBRARY_PATH; \
```

Comment:
I think this is excessive. For my manual testing, I simply set `LD_LIBRARY_PATH=.` and it just works.

The change of `LDFLAGS` wouldn't take effect at this point anyway, because the test scripts being invoked from here assume the tool has already been built.